tests/foreman/cli/test_katello_agent.py

```diff
@@ -18,24 +18,23 @@
 """
 import pytest
 
+from pytest_fixtures.core.contenthosts import register_host_custom_repo
 from robottelo import constants
-from robottelo.cli.factory import setup_org_for_a_custom_repo
-from robottelo.cli.factory import setup_org_for_a_rh_repo
 from robottelo.config import settings
 from robottelo.helpers import InstallerCommand
 
-
 pytestmark = [
     pytest.mark.run_in_one_thread,
-    pytest.mark.skip_if_not_set('clients', 'fake_manifest'),
     pytest.mark.destructive,
+    pytest.mark.tier5,
+    pytest.mark.upgrade,
```

Comment:
This will end up selecting all upgrade tests in present and future. I would recommend adding the flag on selective only tests.

Reply:
yes, I would like to include all these tests of the module for upgrade all-tier run